aspnet/signalr/overview/performance/scaleout-with-redis.md

```diff
@@ -36,7 +36,7 @@ In this tutorial, you will use [Redis](http://redis.io/) to distribute messages
 
 Redis is an in-memory key-value store. It also supports a messaging system with a publish/subscribe model. The SignalR Redis backplane uses the pub/sub feature to forward messages to other servers.
 
-![Image showing arrows going from Redis Server to VMs and then to computers. One arrow labeled "Publish" goes from VMs to Redis Server.](scaleout-with-redis/_static/image1.png)
+![Diagram that shows arrows going from Redis Server to V M and then to computers One arrow labeled Publish goes from V M to Redis Server.](scaleout-with-redis/_static/image1.png)
```

Comment:
Let's split up this sentence.
"Diagram that shows arrows going from Redis Server to V M and then to computers. One arrow labeled Publish goes from V M to Redis Server."